types/example_tuple_test.go

```diff
@@ -19,9 +19,10 @@ package types_test
 import (
 	"fmt"
 
-	"golang.org/x/crypto/blake2b"
-
+	"github.com/centrifuge/go-substrate-rpc-client/scale"
 	. "github.com/centrifuge/go-substrate-rpc-client/types"
+
```

Comment:
import space

Reply:
they are different repositories, I thought that they should go into its own sections.